azure-pipelines.yml

```diff
@@ -36,25 +41,38 @@ stages:
   - stage: system_tests_win64
     dependsOn: []
     pool:
-      name: nimi-bot
+      name: nimibot
```

Comment:
renaming nimi-bot -> nimibot should be separate PRs.
PRs should be single purpose and focused.

Reply:
Sorry @marcoskirsch  I cannot make this as a separate change.

Since new agents are created in  "nimibot", I will not able to test this pipeline changes if I make it as a separate change.

Reply:
ok